package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@oat-sa/eslint-config-tao",
-  "version": "2.0.0",
+  "version": "3.0.0",
```

Comment:
this might be done automatically. Not sure if it's necessary

Reply:
The release action should do it based on [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) (based on your current commits it would go to 2.1.0). If you want to guarantee the major release, it might feel safer to keep this change, but beware of ending up at 3.1.0 after the release 😄 )

Reply:
I wasn't sure if we were using a release action based on conventional commits here. I think my commit should bump the major version. To see the `BREAKING CHANGE` footer in the commit you have to click on the 3 dots. Does it look ok?
![Screenshot 2024-04-24 at 20 23 02](https://github.com/oat-sa/eslint-config-tao/assets/11599360/9421c6b7-8065-42b3-913d-b4c11c4c6e6a)

Reply:
Should I revert my 3.0.0 version change, then?

Reply:
Yes, please remove the version bump. As long as a commit is declared as breaking (either the `!` modifier or the `BREAKING CHANGE` footer), it would trigger a major version increase.

Reply:
ok, done!